lisa/node.py

```diff
@@ -231,11 +250,9 @@ def _initialize(self, *args: Any, **kwargs: Any) -> None:
         if self.is_remote:
             assert (
                 self._connection_info
-            ), "call setConnectionInfo before use remote node"
-            address = f"{self._connection_info.address}:{self._connection_info.port}"
-        else:
-            address = "localhost"
-        self.log.info(f"initializing node '{self.name}' {address}")
+            ), "call set_connection_info before using a remote node"
```

Comment:
How about to inherit from the RemoteNode and overrides the `_initialize`? The subclass doesn't need to be in this repo. So it doesn't need the `sut` flag, and no change on Nodes.

Reply:
I'd have to replicate all the logic before `if not self.sut` (and maintain it), which I'm not willing to. This belongs in the base infra.